EIPS/eip-2831.md

```diff
@@ -0,0 +1,121 @@
+---
+eip: 2831
+title: Transaction Replacement Message Type
+author: Gregory Markou (@GregTheGreek)
+discussions-to: [eth magicians](https://ethereum-magicians.org/t/eip-2831-transaction-replacement-message-type/4448)
```

Comment:
EIPs normally just have a raw link here.  The final EIP renderer may try to pretty this at some point in the future, so I think it is good to keep it as a raw link everywhere.
```suggestion
 discussions-to: https://ethereum-magicians.org/t/eip-2831-transaction-replacement-message-type/4448
```